services/movie.py

```diff
@@ -0,0 +1,39 @@
+from django.db.models import QuerySet
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list = None,
+        actors_ids: list = None
+) -> QuerySet:
+
+    queryset = Movie.objects.all()
+
+    if genres_ids:
+        queryset = queryset.filter(genre__id__in=genres_ids)
+    if actors_ids:
+        queryset = queryset.filter(genre__id__in=actors_ids)
+
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
+
+
+def create_movie(
+        movie_title: str,
+        movie_description: str,
+        /,
+        genres_ids: list = None,
+        actors_ids: list = None
+) -> None:
+
+    movie, _ = Movie.objects.create(
```

Comment:
The `create` method returns a single instance, not a tuple. Remove the comma and underscore (`_`) after `movie`.